src/utils/Utils.h

```diff
@@ -87,4 +87,7 @@ namespace Utils
 	int twosComplement16(uint16_t registerValue);
 
 	void spinWait(uint32_t ms);
+
+    void leaveValidation();
```

Comment:
I think that leaveValidation() and inVaValidation() should be in the ValidationManager rather than in Utils
@jeansimonet what's your opinion?

Reply:
Sorry I didn't get the notification...
Yes, I agree with Olivier.

Reply:
@kylzrnr not sure if you got notified ;)

Reply:
Once you moved the functions, be sure to add a few explanatory comments about how these 2 bits work for enabling/disabling validation mode, and update the ValidationManager comments in header file to be a bit more specific with it does. Thanks!